src/index.js

```diff
@@ -120,8 +143,7 @@ export class WebView extends Component {
     return createElement('iframe', {
       title,
       ref: this.setRef,
-      src: !source.method ? source.uri : undefined,
```

Comment:
This I don't get. Are we eliminating the possibility of having an uri as src within the iframe?

Reply:
No no, all I'm changing is that instead of using the `src` attribute (that only supports `GET`) to specify which page to download and display in the iframe, I use `fetch` in the constructor to download the HTML content and pass that in in the `srcDoc` attribute.

With this change I fixed the `injectedJavascript` prop not working when you just passed in a `uri` in the source, because in that case we didn't have the HTML content at hand where the JS had to be injected, so the `srcDoc` could not be updated with it. It was also inconsistent, because if you specified the `method` and set it to `GET` (which, btw, should be the default value) it used `fetch` and started working, of course. So I thought making it consistently use `fetch` is just going to make it easier to understand and it fixes all the issues as well.

The cons of this is that `srcDoc` is not supported in IE, although I don't see that as a problem. ([Can I use?](https://caniuse.com/iframe-srcdoc))

One other solution would be to this problem is to use the same JS injection method I use in the `injectJavascript` method, but the JS you inject that way is not visible in the DOM so it's harder to debug it. I just found that we need to use that once the page is loaded, otherwise I found code that wasn't properly working with adding JS in `<script>` tags to the DOM.

Reply:
Got it, thanks for the explanation! Let's keep this approach then :)